playwright/fixtures/index.ts

```diff
@@ -35,113 +44,242 @@ export type Fixtures = {
-              {
-                name: "persist:root",
-                value: JSON.stringify({
-                  userContext: {
-                    userReview: {
-                      onboarding: true,
-                      advancedRepository: true,
+      await use({ name: sliceName });
+    },
+
+    /**
+     * Page
+     */
+    page: async ({ browser }, use) => {
+      // Onboard user in Local Storage by default
+      let context = await browser.newContext({
+        storageState: {
+          cookies: [],
+          origins: [
+            {
+              origin: config.use.baseURL,
+              localStorage: [
+                {
+                  name: "persist:root",
+                  value: JSON.stringify({
+                    userContext: {
+                      userReview: {
+                        onboarding: true,
+                        advancedRepository: true,
+                      },
+                      updatesViewed: {
+                        latest: null,
+                        latestNonBreaking: null,
+                      },
+                      hasSeenChangesToolTip: true,
+                      hasSeenSimulatorToolTip: true,
+                      hasSeenTutorialsToolTip: true,
+                      authStatus: "unknown",
+                      lastSyncChange: null,
                     },
-                    updatesViewed: { latest: null, latestNonBreaking: null },
-                    hasSeenChangesToolTip: true,
-                    hasSeenSimulatorToolTip: true,
-                    hasSeenTutorialsToolTip: true,
-                    authStatus: "unknown",
-                    lastSyncChange: null,
-                  },
-                }),
-              },
+                  }),
+                },
+                {
+                  name: "slice-machine_isInAppGuideOpen",
+                  value: "false",
+                },
+              ],
+            },
+          ],
+        },
+      });
+
+      // Prevent user to be onboarded if needed
+      if (!onboarded) {
+        context = await browser.newContext({
+          storageState: {
+            cookies: [],
+            origins: [
               {
-                name: "slice-machine_isInAppGuideOpen",
-                value: "false",
+                origin: config.use.baseURL,
+                localStorage: [],
               },
             ],
           },
-        ],
-      },
-    });
+        });
+      }
+
+      // Create new page object with new context
+      const page = await context.newPage();
+
+      // Logout user by default
+      try {
+        await fs.rm(path.join(os.homedir(), ".prismic"));
+      } catch (error) {
+        // Ignore since it means the user is already logged out
+      }
+
+      // Login user if needed
+      if (loggedIn) {
+        // In CI we define a PRISMIC_URL env variable to fasten the tests
+        let prismicUrl: string | undefined = process.env["PRISMIC_URL"];
```

Comment:
Is `string | undefined` necessary? I'm not sure what the built-in type of `process.env["PRISMIC_URL"]` is, but I generally try to avoid declaring types.

Reply:
Oh good one, removing it!

Reply:
Done